### DISCUSSION TITLE
How to fix autocompletion issue in VS Code?

### DISCUSSION THREAD

**Discussion by sawa-ko** [unocss/unocss#2095] [Q&A]
Currently autocompletion doesn't work on ".vue" files, and doing CTRL + Space doesn't help as it completes only the typed class and doesn't continue to complete the other classes. So you have to re-press that key combination many times and it shouldn't.

I see that my problem in the issues tab has not been solved for a long time, any alternative solution?

My environment is Vue 3 with Nuxt 3 using the Volar extension in VS Code.

**Comment by Rednas83** [comment]
Use the WindiCSS IntelliSension extension, should work very similar
https://github.com/unocss/unocss/issues/1553#issuecomment-1241678436

**Reply by sawa-ko** [reply]
Thanks for your answer. The problem is that all the things that I add to unocss, like presets, for example, that extension doesn't detect them and so the autocompletion is not correct.

Another case is putting branding colors in the themes option, the extension doesn't detect them either.
